docs/api/covidcast_geography.md

```diff
@@ -0,0 +1,137 @@
+  the `jhu-csse` source. These are reported exactly as JHU reports their data;
+  [see below](#fips-exceptions-in-jhu-data).
+* `hrr`: Hospital Referral Region, units designed to represent regional health
+  care markets. There are roughly 300 HRRs in the United States. A map is
+  available
+  [here](https://hub.arcgis.com/datasets/fedmaps::hospital-referral-regions). We
+  report HRRs by their number (non-consecutive, between 1 and 457).
+* `msa`: Metropolitan Statistical Area, as defined by the Office of Management
+  and Budget. The Census Bureau provides [detailed definitions of these
+  regions](https://www.census.gov/programs-surveys/metro-micro/about.html). We
+  report MSAs by their CBSA ID number.
+* `dma`: Designated Market Areas represent geographic regions with their own
+  media markets, as [defined by
+  Nielsen](https://www.nielsen.com/us/en/intl-campaigns/dma-maps/).
+* `state`: The 50 states, identified by their two-digit postal abbreviation (in
+  lower case). Estimates for Puerto Rico are available as state `pr`; Washington, D.C. is available as state `dc`.
+
+Some signals are not available for all `geo_type`s, since they may be reported
+from their original sources with different levels of aggregation.
+
+1. toc
+{:toc}
+
+## Small Sample Sizes and "Megacounties"
+
+Most sources do not report the same amount of data for every county; for
+example, the survey sources rely on survey responses each day, and many counties
+may have comparatively few survey responses. We do not report individual county
+estimates when small sample sizes would make estimates unreliable or would allow
+identification of respondents, violating privacy and confidentiality agreements.
+Additional considerations for specific signals are discussed in the [source and
+signal documentation](covidcast_signals.md).
+
+In each state, we collect together the data from all counties with insufficient
+data to be individually reported. These counties are combined into a single
+"megacounty". For example, if only five counties in a state have sufficient data
+to be reported, the remaining counties will form one megacounty representing the
+rest of that state. As sample sizes vary from day to day, the counties composing
+the megacounty can vary daily; the geographic area covered by the megacounty is
+simply the state minus the counties reported for that day.
+
+Megacounty estimates are reported with a FIPS code ending with 000, which is
+never a FIPS code for a real county. For example, megacounty estimates for the
+state of New York are reported with FIPS code 36000, since 36 is the FIPS code
+prefix for New York.
+
+
+## FIPS Exceptions in JHU Data
+
+At the County (FIPS) level, we report the data _exactly_ as JHU reports their
+data, to prevent confusing public consumers of the data. JHU FIPS reporting
+matches that used in the other signals, except for the following exceptions.
+
+### New York City
+New York City comprises of five boroughs:
```

Comment:
```suggestion
New York City comprises five boroughs:
```

You could also say "is composed of" "is made up of" or "contains" here; "comprises" is always tricky